sunpy/map/mapbase.py

```diff
@@ -1675,13 +1683,11 @@ def peek(self, draw_limb=False, draw_grid=False,
         if draw_limb:
             self.draw_limb(axes=axes)
 
-        if isinstance(draw_grid, bool):
-            if draw_grid:
+        if draw_grid:
+            if grid_spacing is None:
```

Comment:
Maybe this is ignorant question, but what's the reasoning behind adding the `if-else` block here? Could it not just be 
```python
self.draw_grid(axes=axes, grid_spacing=grid_spacing)
```
independent of whether `grid_spacing` is `None`? Is there some reason to not pass `None` to `draw_grid`?

Reply:
Passing `grid_spacing=None` to `draw_grid()` isn't supported according to the docstring, and I think I tried this a while ago and it indeed was not supported.